src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -3547,6 +3557,27 @@ private void EmitConditionalOperator(BoundConditionalOperator expr, bool used)
             }
 
             _builder.MarkLabel(doneLabel);
+
+            static bool hasIntegralValueZeroOrOne(BoundExpression expr, out bool one)
```

Comment:
```suggestion
            static bool hasIntegralValueZeroOrOne(BoundExpression expr, out bool isOne)
```

Consider an `is` prefix here as it affirms it's a boolean value.